ts/Dashboards/README.md

```diff
@@ -0,0 +1,127 @@
+# Highcharts Dashboards #
+A JavaScript library for interactive dashboards.
+
+## Installation
+There are multiple different ways how you can import this package.
+Among the most popular are:
+* Using npm
+Run the following command in your terminal:
+```bash
+    npm install <PACKAGE-NAME>
+```
+Then import the package in your project:
+``` JS
+    import Dashboards from '<PACKAGE-NAME>';
+```
+* Importing as a script
+```html
+    <script src="https://code.highcharts.com/dashboards.js"></script>
+ ```
+
+To fully utilize the Dashboards potential, there might be a need to load additional modules.  
+In the [installation documentation](https://highcharts.com/docs/dashboards/installation), you can find more information on how to do that and what are the other ways of importing the Dashboards.
+
+## Components
+Each dashboard is built with different components. You can add the most basic `HTMLComponent` where you can add some text, image etc.
+To create a chart you can add `HighchartsComponent`. If you would like to show your data in a tabular way use the `DataGridComponent`. Or use the `KPIComponent` to highlight some individual numbers/indications.
+
+To properly show the component you have to declare the `id` of a cell where it should be placed and the `type` of that component.
+An example of how the component might look like:
+``` JS
+    {
+        type: 'Highcharts',
+        cell: 'cell-id-2',
+        chartOptions: {
+            series: [{
+                type: 'pie',
+                data: [1, 2, 3, 2, 3]
+            }]
+        }
+    }
+```
+In the [component documentation](https://highcharts.com/docs/dashboards/types-of-components), you can find more information on how each one of them works and what can be configured.
+
+## Working with Data
+
+You can import your data from multiple sources for example CSV and Google Spreadsheet. They are going to be handled by the Stores and distributed as DataTables.  
+
+More about this concept in the [Data documentation](https://highcharts.com/docs/dashboards/data).
+
+## GUI
+The GUI is a part of the dashboard that allows you to create a layout of the dashboard. You can add rows and cells to the layout. The layout is a grid where you can place your components.  
+An example of how the GUI configuration might look like:
+``` JS
```

Comment:
not really an issue, but why big JS?

Reply:
I guess it is an abbreviation. Fun fact: You could also write JavaScript / Shell / TypeScript.

Reply:
With that, the code is colored nicely ;)